test/powershell/Modules/Microsoft.PowerShell.Management/Set-Service.Tests.ps1

```diff
@@ -195,6 +195,46 @@ Describe "Set/New-Service cmdlet tests" -Tags "Feature", "RequireAdminOnWindows"
         }
     }
 
+    It "Remove-Service can remove a service" {
+        try {
+            $servicename = "testremoveservice"
+            $parameters = @{
+                Name           = $servicename;
+                BinaryPathName = "$PSHOME\powershell.exe"
+            }
+            $service = New-Service @parameters
+            $service | Should Not BeNullOrEmpty
+            Remove-Service -Name $servicename
+            $service = Get-Service -Name $servicename -ErrorAction SilentlyContinue
+            $service | Should BeNullOrEmpty
+        }
+        finally {
+            Get-CimInstance Win32_Service -Filter "name='$servicename'" | Remove-CimInstance -ErrorAction SilentlyContinue
+        }
+    }
+
+    It "Remove-Service can accept pipeline input of a ServiceController" {
+        try {
+            $servicename = "testremoveservice"
+            $parameters = @{
+                Name           = $servicename;
+                BinaryPathName = "$PSHOME\powershell.exe"
+            }
+            $service = New-Service @parameters
+            $service | Should Not BeNullOrEmpty
+            Get-Service -Name $servicename | Remove-Service -ErrorAction SilentlyContinue
```

Comment:
I don't think you want `-ErrorAction SilentlyContinue` there in case something fails